libraries/ms-common/inc/log.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <stdio.h>
+
+typedef enum {
+  LOG_LEVEL_DEBUG = 0,
+  LOG_LEVEL_WARN,
+  LOG_LEVEL_FAULT,
+  LOG_LEVEL_NONE, // Invalid log level
```

Comment:
The definition for invalid enums I have been using is `NUM_LOG_LEVEL`. Not sure if we care?